server/app/views/admin/programs/ProgramApplicationListView.java

```diff
@@ -251,7 +253,9 @@ private FormTag renderSearchForm(
                     downloadButton,
                     makeSvgTextButton("Filter", Icons.FILTER_ALT)
                         .withClass(ButtonStyles.SOLID_BLUE_WITH_ICON)
-                        .withType("submit")));
+                        .withType("submit"),
+                  a().withHref(CLEAR_REDIRECT)
```

Comment:
**suggestion (blocking):** use the reverse router https://www.playframework.com/documentation/2.8.x/JavaRouting#Reverse-routing